snp_motion_planning/src/planning_server.cpp

```diff
@@ -576,13 +616,67 @@ class PlanningServer
 
     RCLCPP_INFO_STREAM(node_->get_logger(), res->message);
   }
+  void freespaceMotionPlanCallback(const snp_msgs::srv::GenerateFreespaceMotionPlan::Request::SharedPtr req,
+                                   snp_msgs::srv::GenerateFreespaceMotionPlan::Response::SharedPtr res)
+  {
+    try
+    {
+      RCLCPP_INFO_STREAM(node_->get_logger(), "Received freespace motion planning request");
+
+      tesseract_common::ManipulatorInfo manip_info;
+      manip_info.manipulator = req->motion_group;
+      manip_info.tcp_frame = req->tcp_frame;
+      manip_info.working_frame = req->mesh_frame;
```

Comment:
Not sure that we need a `working_frame`, but I can't remember the requirements of tesseract for this

Reply:
Apparently an assert fails if `working_frame` or `tcp_frame` isn't populated